<ui version="4.0" >
 <class>AssignDialog</class>
 <widget class="QDialog" name="AssignDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="toolTip" >
        <string>Номер модуля</string>
       </property>
       <property name="text" >
        <string>Модуль:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="le1" >
       <property name="toolTip" >
        <string>Номер модуля</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="toolTip" >
        <string>Номер слота опроса модуля</string>
       </property>
       <property name="text" >
        <string>Слот:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="le2" >
       <property name="toolTip" >
        <string>Номер слота опроса модуля</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="toolTip" >
        <string>Номер сигнала в слоте опроса модуля</string>
       </property>
       <property name="text" >
        <string>Номер:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="le3" >
       <property name="toolTip" >
        <string>Номер сигнала в слоте опроса модуля</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Заполнение ячеек</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="cbox_fill_row" >
        <property name="text" >
         <string>по строкам:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cb_fill_row" />
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="cbox_fill_column" >
        <property name="text" >
         <string>по стлобцам:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="cb_fill_column" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Формат данных:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_format" >
     <item>
      <property name="text" >
       <string>Десятичный</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Десятичный беззнаковый</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Шестнадцатиричный</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Двоичный</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Перечислитель:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_ss" />
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le1</tabstop>
  <tabstop>le2</tabstop>
  <tabstop>le3</tabstop>
  <tabstop>cb_format</tabstop>
  <tabstop>le_ss</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AssignDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AssignDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
